test/services/ip-address/ipAddress.service.test.ts

```diff
@@ -0,0 +1,29 @@
+import { assert } from 'chai'
+import { Request } from 'express'
+import { generateRequest } from '../../fixtures/http.fixtures'
+
+import IpAddressService from 'app/services/ip-address/ipAddress.service'
+
+describe('IpAddressService', () => {
+  let service: IpAddressService
+
+  let req: Request
+
+  const IP_ADDRESS = '127.0.0.1'
+
+  describe('getIpAddress', () => {
+
+    it('should respond with the IP address from the provided request', () => {
+
+      req = generateRequest()
+
+      req.headers = {'x-forwarded-for' : IP_ADDRESS}
+
+      service = new IpAddressService()
```

Comment:
the service instantiation can be done at the top of this file rather than in the unit test:
```
const service: IpAddressService = new IpAddressService()
```